drivers/usb/udc/udc_numaker.c

```diff
@@ -1016,6 +1050,15 @@ static int numaker_xusbd_ep_xfer_user_dma(struct numaker_xusbd_ep *ep_cur, uint8
 			       : 0;
 	base->DMACTL |= USB_EP_GET_IDX(ep_cur->addr) << HSUSBD_DMACTL_EPNUM_Pos;
 
+	/* Cache coherency */
+#if !defined(CONFIG_UDC_BUF_FORCE_NOCACHE)
```

Comment:
Why would be the option UDC_BUF_FORCE_NOCACHE enabled?

Reply:
Removed. `sys_cache_data` functions can cover no-cache case.